contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol

```diff
@@ -146,7 +165,7 @@ contract VRFCoordinatorV2Mock is VRFCoordinatorV2Interface {
     uint16 _minimumRequestConfirmations,
     uint32 _callbackGasLimit,
     uint32 _numWords
-  ) external override onlyValidConsumer(_subId, msg.sender) returns (uint256) {
+  ) external override nonReentrant onlyValidConsumer(_subId, msg.sender) returns (uint256) {
```

Comment:
I believe the nonReentrant guard is also applied to `cancelSubscription`, `migrate`, and removeConsumer` in the VRFV2+ coordinator, although I'm not sure if all those functions are applied in the mockfile.

Reply:
I've added the nonReentrant guard for `cancelSubscription` and `removeConsumer` in VRFCoordinatorV2Mock.sol.
I couldn't find migrate in VRFCoordinatorV2.sol or VRFCoordinatorV2Mock.sol.